composables/useUiHelpers/index.ts

```diff
@@ -0,0 +1,19 @@
+type Category = {
+  categoryId: Number
+  categoryCode: String
+}
+
+interface uiHelpersReturnType {
```

Comment:
Should move type declarations for composables into compsables/types

Reply:
resolved.